spec/etcd_env_spec.rb

```diff
@@ -22,12 +22,19 @@
   it 'should pass environment variables to scripts' do
     `etcdctl set /test/FOO ham`
     `etcdctl set /test/BAR cheese`
-    expect(`./exe/etcd-env /test 'echo $FOO $BAR'`.chomp).to eq 'ham cheese'
+    expect(`./exe/etcd-env /test -- 'echo $FOO $BAR'`.chomp).to eq 'ham cheese'
```

Comment:
I keep that for later. I want to have it working first with several namespaces first as we almost always get variables from more than one for now.